### PR TITLE
DOC: 1.1.3 release date

### PR DESCRIPTION
**Pull request by simonjayhawkins**
removed mention of py39 support @jreback

### CODE REVIEW

**Comment by jreback** [comment]
@simonjayhawkins we do support 3.9; ideally have the wheels, if not then that's ok (though IIRC they are elsewhere being worked on). don't remove it.

File: doc/source/whatsnew/v1.1.3.rst [hunk 1]
```diff
@@ -10,21 +10,6 @@ including other versions of pandas.
 
 .. ---------------------------------------------------------------------------
 
-Enhancements
```
Comment (jreback) [inline doc/source/whatsnew/v1.1.3.rst:13]: revert this

Comment (simonjayhawkins) [reply doc/source/whatsnew/v1.1.3.rst]: done

**Comment by jreback** [comment]
lgtm.

**Pull request merged**